snippets/avoid_fixture_name_collision.rb

```diff
@@ -0,0 +1,57 @@
+if __FILE__ =~ /^snippets/
```

Comment:
Just a note that we have no support for snippets in `main`, the code to run them is in `rails-6-1-dev` only.
But if CI passes on `rails-6-1-dev`, the patch should most possibly work for `main`, too.

Reply:
Is it easy-ish to backport that support?

Reply:
Should be easy. Looks like cherry-picking https://github.com/rspec/rspec-rails/commit/040fecc98894de42a433d94a15d19f07434b77f8 and accepting the removed lines from `Gemfile` should do it.